frontend/encore/bootstrap.rst

```diff
@@ -1,18 +1,24 @@
-Using Bootstrap CSS & JS
-========================
+Using Bootstrap 4 CSS & JS
```

Comment:
Let's revert this change and remove the `4` version number. This article will be perpetually updated when new versions of Bootstrap are published, so there's no need to update the title.